hack/annotate.sh

```diff
@@ -9,10 +9,28 @@ base="$( dirname "${BASH_SOURCE[0]}" )/.."
 function annotate() {
 	local namespace="$1"
 	local name="$2"
+	local private="${4:-}" # empty string by default
 	local conf="${base}/core-services/release-controller/_releases/release-$3"
+
+	if [[ -n "${private}" ]]; then
+  	conf="${base}/core-services/release-controller/_releases/priv/release-$3"
+  fi
+
 	if [[ -s "${conf}" ]]; then
 		echo "${conf}"
 		jq . <"${conf}"
+
+  	# If this is a configuration for a private release controller, enforce that all ProwJob
+  	# names include "priv". This attempts to ensure that no on introduce a ProwJob without
+  	# "hidden: true" to the verfication steps of embargoed release payloads.
+  	if [[ -n "${private}" ]]; then
+      local nonpriv_hits=$(cat ${conf} | jq -r '.verify | keys[] as $k | (.[$k] | .prowJob.name)' | grep -v priv)
+      if [[ -n "${nonpriv_hits}" ]]; then
+        echo "${conf} contains prowJob name 'priv' substring ; Please use naming convention to ensure embargoed releases are not tested publicly."
```

Comment:
This looks misleading? Says "... contains 'priv' substring", but the check is opposite? We are searching for things that are missing `priv` and shoudl warn on them, correct?

Reply:
Nice catch. Next push include '*without* "priv" substring'